core/client/db/src/lib.rs

```diff
@@ -110,6 +110,12 @@ impl<B: BlockT> Drop for RefTrackingState<B> {
 	}
 }
 
+impl<Block: BlockT> std::fmt::Debug for RefTrackingState<Block> {
+	fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+		write!(f, "Block {:?}", self.parent_hash)
```

Comment:
https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html ?

Otherwise you don't know where `Block` belongs to.

Reply:
Type name is not that important for debugging here